apps/server/src/migrations/mikro-orm/Migration20241030126666.ts

```diff
@@ -1,6 +1,6 @@
 import { Migration } from '@mikro-orm/migrations-mongodb';
 
-export class Migration202410041210124 extends Migration {
+export class Migration20241030126666 extends Migration {
```

Comment:
renamed migration?

Reply:
I tried to fix the class name because it doesn't match the file name. But yesm it's maybe safer to leave it as it is. Will change it back.